datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/policy/GetGrantedPrivilegesResolver.java

```diff
@@ -44,9 +50,33 @@ public CompletableFuture<Privileges> get(final DataFetchingEnvironment environme
     if (context.getAuthorizer() instanceof AuthorizerChain) {
       DataHubAuthorizer dataHubAuthorizer =
           ((AuthorizerChain) context.getAuthorizer()).getDefaultAuthorizer();
-      List<String> privileges = dataHubAuthorizer.getGrantedPrivileges(actor, resourceSpec);
+
       return GraphQLConcurrencyUtils.supplyAsync(
-          () -> Privileges.builder().setPrivileges(privileges).build(),
+          () -> {
+            try {
+              PolicyEngine.PolicyGrantedPrivileges evalResult =
+                  dataHubAuthorizer.getGrantedPrivileges(actor, resourceSpec);
+
+              List<PolicyEvaluationDetail> evaluationDetailList = null;
+
+              if (input.getIncludeEvaluationDetails()
+                  && PolicyAuthUtils.canManagePolicies(context)) {
```

Comment:
Update the release notes to call out the breaking change. This graphQL query now requires `Manager Policies` permissions

Reply:
At minimum this should be called out in the docs for `entity.graphQL` that if the evaluationDetails are requested the user needs the permissions.